blocks/library/embed/index.js

```diff
@@ -76,6 +81,7 @@ registerBlockType( 'core/embed', {
 		constructor() {
 			super( ...arguments );
 			this.doServerSideRender = this.doServerSideRender.bind( this );
+			this.isPreviewBlacklisted = this.isPreviewBlacklisted.bind( this );
```

Comment:
This is getting removed now that the noPreview domains are getting slightly reworked.